### ISSUE TITLE
Improvement: Haptic Feedback

### ISSUE THREAD

**Issue by danielsogl** [fluster/fluster-app#5]
Sounds like a gimmick but haptic feedback improves the user experience (imao). For example toggeling the notifications from on to off. I also like haptic feedback when something goes wrong for example entering an false password.

**Comment by peterpeterparker** [comment]
Good idea, also I would learn something because I don't now that subject at all ;)

I might prioritize some other subjects but an interesting subject, thx for the input

**Comment by danielsogl** [comment]
I think this feature is more typical for ios devices because of the Apple Taptic Engine.
